

Header header

float32 batteryVoltage
float32 servoTemperature

string state

ServoDiag[] servos
